Task_11/index.html

```diff
@@ -0,0 +1,308 @@
+            <li>Yesterday</li>
+          </ul>
+
+          <div class="box">
+            <h3>Timeline Info</h3>
+            <p>With an <strong>average</strong> timeline set up, our expert translator can take a reasonable amount of time perfecting tour translation</p>
+          </div>
+
+        </div>
+
+      </fieldset>
+      
+      <fieldset class="signUp">
+
+        <h2>Sign up and choose a payment method</h2>
+
+        <p>Already have an account with us? <a href="#">Sign in</a>.</p>
+
+        <div class="personalInfo">
+
+          <p>
+            <label for="name">Full Name</label>
+            <input type="text" name="name" class="box" id="name" pattern="[A-Za-z' -]+" minlength="2" maxlength="50" required placeholder=" ">
+          </p>
+
+          <p>
+            <label for="email">Email Adress</label>
+            <input type="email" name="email" class="box" id="email"  pattern="[A-Za-z0-9._-]+@[a-z]+\.[a-z]{2,4}$" required placeholder=" ">
+          </p>
+
+          <p>
+           <label for="password">Choose a Password</label>
+            <input type="password" name="password" class="box" id="password" required pattern="(?=.*[0-9])(?=.*[a-z])(?=.*[A-Z]).{8,}" title="Must contain at least one number and one uppercase and lowercase letter, and at least 8 or more characters" placeholder=" ">
+          </p>
+
+        </div>
+
+        <div class="choicesWrapper paymentType">
+
+          <input hidden type="radio" id="visa" name="payment" value="visa">
+          <label for="visa" class="box"></label>
+          
+          <input hidden type="radio" id="payPal" name="payment" value="payPal" checked>
+          <label for="payPal" class="box"></label>
+    
+          <input hidden type="radio" id="amazon" name="payment" value="amazon">
+          <label for="amazon" class="box"></label>
+
+        </div>
+
+        <div class="choicesWrapper paymentInfo">
+
+          <p>
+            <label for="number">Card Number</label>
+            <input type="text" name="cardNumber" class="box" id="number" pattern="[0-9]{16}" required>
+          </p>
+
+          <p>
+            <label for="date">Expiry Date</label>
+            <input type="date" name="expiryDate" class="box" id="date" required>
+          </p>
+
+          <p>
+            <label for="code">Code</label>
+            <input type="text" name="code" class="box" id="code" pattern="[0-9]{3}" required>
+          </p>
+
+        </div>
+
+      </fieldset>
+
+      <div class="summingWrapper">
+
+        <input type="checkbox" name="agreement" value="agree" id="agreement" required>
+        <label for="agreement">I agree to the <a href="#">Terms &amp; Conditions</a></label>
+
+        <input type="submit" class="box" name="placeOrder" value="Place Order">
+
+        <button class="box printButton" type="button">Print a quote</button>
+
+      </div>
+
+    </form>
+
+  </main>
+  
+  <footer>
+    
+    <div class="whatWeDo">
```

Comment:
Semantic element for such block is `<legend>`

Reply:
Why should it be a `<legend>`?

Reply:
oh confused, sec

Reply:
now its ok.!

Reply:
Could you explain why `<aside>`?
It is a little bit tricky for me...